examples/sycl/bf_knn_classification_batch.py

```diff
@@ -48,6 +59,16 @@ def to_numpy(data):
         pass
     return data
 
+# Common code for both CPU and GPU computations
+def compute(train_data, train_labels, predict_data, nClasses):
+    # Create an algorithm object and call compute
+    train_algo = d4p.bf_knn_classification_training(nClasses=nClasses)
+    train_result = train_algo.compute(train_data, train_labels)
+
+    # Create an algorithm object and call compute
+    predict_algo = d4p.bf_knn_classification_prediction()
```

Comment:
BF kNN doesn't work without classes specification on prediction stage
```suggestion
    predict_algo = d4p.bf_knn_classification_prediction(nClasses=nClasses)
```

Reply:
Thanks!